plugins/redfish/tests/redfish.py

```diff
@@ -18,9 +18,6 @@
 
 
 def _failure(msg: str, status=400):
-    res = {
-        "error": {"message": msg},
-    }
     return Response(response=json.dumps(msg), status=401, mimetype="application/json")
```

Comment:
I actually think this should have been json.dumps(res) -- does that look plausible?

Reply:
Yeah, looks more reasonable to me.  Will switch.
```
>>> from flask import Response
>>> import json
>>> msg='foo'
>>> Response(response=json.dumps(msg), status=401, mimetype="application/json")
<Response 5 bytes [401 UNAUTHORIZED]>
>>> res = {
...         "error": {"message": msg},
...     }
>>> Response(response=json.dumps(res), status=401, mimetype="application/json")
<Response 29 bytes [401 UNAUTHORIZED]>
```